fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/metric/DruidMetricName.java

```diff
@@ -0,0 +1,28 @@
+package com.yahoo.wiki.webservice.data.config.metric;
+
+import com.yahoo.bard.webservice.data.config.names.FieldName;
+import com.yahoo.bard.webservice.data.time.TimeGrain;
+
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Created by kevin on 3/7/2017.
+ */
+public class DruidMetricName implements FieldName {
+    private String lowerCaseName;
+
+    public DruidMetricName(String lowerCaseName) {
+        this.lowerCaseName = lowerCaseName.toLowerCase(Locale.ENGLISH);
```

Comment:
clearly the incoming parameter should not be called lowerCaseName, since that's not being enforced or expected.  

Also, constructor javadoc indicated, with parameter description.